src/bin/drone.rs

```diff
@@ -127,10 +133,6 @@ fn main() {
                     }
                     Ok(())
                 })
-                .and_then(|()| {
```

Comment:
If you are cleaning up the messaging, probably should remove the following `or_else` as well. It is redundant.

Reply:
✔️